rasa/utils/train_utils.py

```diff
@@ -316,25 +335,26 @@ def create_tf_dataset(
     batch_size: Union["tf.Tensor", int],
     batch_strategy: Text = "sequence",
     shuffle: bool = False,
+    label_key: Text = "labels",
```

Comment:
I don't think, we need a default label here

Reply:
already changed, but not committed yet